Readme.md

```diff
@@ -46,7 +55,11 @@ dependencies {
 }
 ````
 
-To install the dependencies follow the  installation guides in the Readme documentation of the project. Make sure to publish it as a local Maven repository:
+Note that the version number (in the example above it is set to `1.0.0`) might differ depending on the
+version of the 5GMS Media Session Handler.
```

Comment:
Should this be "depending on the version of the 5GMS Common android Library"?

Reply:
It is a bit midleading but what I meant is that a certain version of the Media Session Handler requires a certain version of the Common Library to run. For instance, version 1.2.0 of the Media Session Handler can not run with version 1.0.0 of the Common Library.